bin/node/cli/tests/common.rs

```diff
@@ -14,7 +14,14 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-use std::{process::{Child, ExitStatus}, thread, time::Duration};
+#![cfg(unix)]
+#![allow(dead_code)]
```

Comment:
What is dead inside here? :P

Reply:
I don't know. I think common.rs is executed on its own...

Reply:
So everything looks like dead code when it's compiled